phc-rails/app/controllers/api/v1/accounts_controller.rb

```diff
@@ -1,11 +1,14 @@
 class Api::V1::AccountsController < ApplicationController
-  respond_to :json
   skip_before_action :verify_authenticity_token
   before_action :verify_security_token
 
   def show
-    sf_id = params[:sf_id]
-    respond_with Account.find_by sf_id: sf_id
+    account = Account.find_by(sf_id: params[:sf_id])
+    if !account.nil?
+      render json: account
+    else
+      api_message_response(401, "Account with that id does not exist.")
```

Comment:
api_message_response is a helper method I defined in the `ApplicaitionController`. It's inherited by all other controllers and helps reduce repetition of code.

See https://github.com/calblueprint/phc/pull/338/files#diff-a0c74587805b9556430a52e4c902f2ccR33.